guides/collectionOrder.md

```diff
@@ -2,35 +2,83 @@
 
 # How to create a collection offer (maker bid with collection strategy)
 
-A collection order is just a maker bid order using the `collection strategy`.
+The code snippet below is an example of how to create a maker bid using the `@looksrare/sdk-v2` library.
+
+> A collection order is just a maker bid order using the `StrategyType.collection`.
+
+The main steps are:
+
+1. Initialize a LooksRare class instance by providing the chain id, [RPC provider](https://docs.ethers.io/v5/api/providers/) and a [signer](https://docs.ethers.io/v5/api/signer/).
+2. Use the `createMakerCollectionOffer` method to create a maker ask with the parameters of your order.
+3. Check and grant necessary approvals for transferring assets.
+4. Sign the maker bid order with `signMakerOrder` method.
+
+> The `orderNonce` has to be retrieved via our Public API, see [get order nonce](https://looksrare.dev/v2/reference/getordernonce).
+
+Here is an example:
 
 ```ts
 import { ethers } from "ethers";
 import { LooksRare, SupportedChainId, CollectionType, StrategyType } from "@looksrare/sdk-v2";
 
 const lr = new LooksRare(SupportedChainId.MAINNET, provider, signer);
 
-const { makerAsk, isCurrencyApproved } = await lr.createMakerCollectionOffer({
+const { makerBid, isCurrencyApproved } = await lr.createMakerCollectionOffer({
   collection: "0x0000000000000000000000000000000000000000", // Collection address
   collectionType: CollectionType.ERC721,
   subsetNonce: 0, // keep 0 if you don't know what it is used for
   orderNonce: 0, // You need to retrieve this value from the API
-  endTime: Math.floor(Date.now() / 1000), // If you use a timestamp in ms, the function will revert
+  endTime: Math.floor(Date.now() / 1000) + 86400, // If you use a timestamp in ms, the function will revert
   price: ethers.utils.parseEther("1"), // Be careful to use a price in wei, this example is for 1 ETH
   amounts: [1], // Use it for listing multiple ERC-1155 (Optional, Default to [1])
   startTime: Math.floor(Date.now() / 1000), // Use it to create an order that will be valid in the future (Optional, Default to now)
 });
 
 // Approve spending of the currency used for bidding
 if (!isCurrencyApproved) {
+  // Send approval transaction on-chain
   const tx = await lr.approveErc20(lr.addresses.WETH);
+
+  // Wait for the transaction to be processed
   await tx.wait();
 }
 
 // Sign your maker order
-const signature = await lr.signMakerOrder(makerAsk);
+const signature = await lr.signMakerOrder(makerBid);
 ```
 
+# How to send the order via our Public API
```

Comment:
I believe this should be in the api doc instead of the SDK doc, and we can add a link there.
If you really want to this in the sdk, I suggest to create a new file `postMakerOrder.md` in the `guide` folder, copy this doc inside, and then add a link here (and in other files, so you don't duplicate everywhere)